server/app/views/support/UnsupportedBrowserView.java

```diff
@@ -0,0 +1,58 @@
+package views.support;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.h1;
+import static j2html.TagCreator.img;
+
+import j2html.tags.ContainerTag;
+import javax.inject.Inject;
+import play.twirl.api.Content;
+import views.BaseHtmlView;
+import views.HtmlBundle;
+import views.applicant.ApplicantLayout;
+import views.style.StyleUtils;
+import views.style.Styles;
+
+public class UnsupportedBrowserView extends BaseHtmlView {
+
+  private final ApplicantLayout applicantLayout;
+
+  @Inject
+  public UnsupportedBrowserView(ApplicantLayout applicantLayout) {
+    this.applicantLayout = checkNotNull(applicantLayout);
+  }
+
+  public Content render() {
+    HtmlBundle bundle = applicantLayout.getBundle();
+
+    bundle.setTitle("The browser you are currently using is not supported");
+    ContainerTag container =
+        div().withClasses(Styles.CONTAINER, Styles.MX_AUTO, Styles.MAX_W_XL, Styles.P_4);
+    container.with(
```

Comment:
Not blocking: Feel free to chain these since they're unconditionally added.

e.g.:

```
DivTag container =
  div()
    .withClasses(...)
    .with(
      h1(...),
      div(...)
    );
```

Same for the below initialization of the `browsers` var. If so inclined, you can inline all of this into one big initialization, but that's definitely a style preference.

Reply:
Done.